src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj

```diff
@@ -31,7 +31,7 @@
     <Compile Include="System\MarshalByRefObjectTest.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)'=='netcoreapp'">
-    <Compile Include="System\BitConverter.netcoreapp.cs" />
```

Comment:
It would be better to use `!= 'nefx'` condition. We want the tests enabled for uap and uapaot as well.